packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -352,14 +304,9 @@ test.describe('PieCookieBanner - Component tests', () => {
 
     test.describe('`cookieStatementLink` prop', () => {
         test.describe('when not populated', () => {
-            test('should set a default cookie statement link of empty string within the banner description', async ({ mount }) => {
+            test('should set a default cookie statement link of empty string within the banner description', async () => {
                 // Arrange & Act
-                await mount(
-                    PieCookieBanner,
-                    {
-                        props: {},
-                    },
-                );
+                await pieCookieBannerComponent.load({ cookieStatementLink: '' });
```

Comment:
Is the output of this still the same without passing the prop value in? The test reads like it should be testing the default value rather than what happens if you pass in an empty string.

Reply:
No, this then defaults back. We are testing the behaviour of passing an empty string into the link.